elastic/kibana
# [Console] Fix leaky mappings subscription

## Summary

Fixes the first issue mentioned here: https://github.com/elastic/kibana/issues/45032

Also fixes unmount never being called.

## How to review

1. Start Kibana + go to Console
2. Check network tab (may be useful to set `const POLL_INTERVAL = 60000;` to a `10000` to see the requests at a higher rate).
3. Navigate away from Console.
4. Keep an eye on the network tab for any `proxy?_alias...` requests firing.

## Release Note

Fixed a subscription in Console that didn't clear even after navigating away from the app.
